plugins/org.jkiss.dbeaver.ui.app.standalone/src/org/jkiss/dbeaver/ui/app/standalone/DBeaverCommandLine.java

```diff
@@ -176,6 +180,24 @@ static boolean executeCommandLineCommands(@Nullable CommandLine commandLine, @Nu
             }
         }
 
+        {
+            //Set configuration file for SystemVariableResolver
+            String file = commandLine.getOptionValue(PARAM_CONFIGURATION_FILE);
+            if (!CommonUtils.isEmpty(file)) {
+                Properties properties = new Properties();
+                try {
+                    InputStream stream = new FileInputStream(file);
+                    properties.load(stream);
+                    SystemVariablesResolver.INSTANCE.setConfigurationFilePath(properties);
+                } catch (Exception e) {
+                    log.error("Error parsing command line: " + e.getMessage());
```

Comment:
Just pass the exception as a separate parameter instead of concatenating its message into a log message.